docs/source/manual/validation.rst

```diff
@@ -0,0 +1,254 @@
+.. _man-validation:
+
+#####################
+Dropwizard Validation
+#####################
+
+.. highlight:: text
+
+.. rubric:: Dropwizard comes with a host of validation tools out of the box to allow endpoints to return meaningful error messages when constraints are violated. Hibernate_ is packaged with Dropwizard, so what can be done in Hibernate, can be done with Dropwizard.
+
+.. _Hibernate: http://hibernate.org/validator/
+
+.. _man-validation-validations:
+
+Validations
+===========
+
+Almost anything can be validated on resource endpoints. To give a quick example, the following
+endpoint doesn't allow a null or empty ``name`` query parameter.
+
+.. code-block:: java
+
+    @GET
+    public String find(@QueryParam("name") @NotEmpty String arg) {
+        // ...
+    }
+
+If a client sends an empty or nonexistant name query param, Dropwizard will respond with a 400 error
```

Comment:
Maybe we could change 400 to  ``400 Bad Request``? I think, it's important to highlight it.